ledger_secure_sdk_sys/src/c/src.c

```diff
@@ -277,6 +279,21 @@ void c_boot_std() {
     io_seproxyhal_spi_send(c, 4);
 #endif
 
+#ifndef TARGET_NANOS
+    // Warn UX layer of io reset to avoid unwanted pin lock
+    memset(&G_ux_params, 0, sizeof(G_ux_params));
+    G_ux_params.ux_id = BOLOS_UX_IO_RESET;
+
+    // If the app has just been booted from the UX, multiple os_ux calls may be necessary
+    // to ensure UX layer has take the BOLOS_UX_IO_RESET instruction into account.
+    for (uint8_t i = 0; i < 2; i++) {
```

Comment:
Why 2 ?

Reply:
Good question, just a copy/paste from the C SDK: https://github.com/LedgerHQ/ledger-secure-sdk/blob/55afb4121b299635fae561239e44d2d5a60e3780/src/os_io_seproxyhal.c#L428